### DISCUSSION TITLE
originalen USB Optolink-Adapter

### DISCUSSION THREAD

**Discussion by hansmusterman** [openv/openv#542] [General]
Hallo und guten Tag,

ich habe ein originalen USB Optolink-Adapter mit meiner Vitotronic 200 KW2 verbunden.
Auf einem Raspberry habe ich das vcontrold_0.98.10-176_armhf.deb installiert.
vcontrold.xml und vito.xml für die 200 KW2 aus dem Beispiel genommen, darin auch meine IP angepasst.
wenn ich jetzt z.B. im Telnet getTempA eingebe kommt das:

vctrld>getTempA
ERR: read timeout
Error in wait, terminating
Error executing getTempA

Muss man den USB Adapter noch konfigurieren? Oder woran kann es liegen?

Danke
HM

**Comment by Bavarese** [comment]
Ich würde davon ausgehen, daß die XML-Dateien von Vcontrol / Vcontrold nicht zur vorhandenen Heizung passen.
Mußte ungefähr 50 % der Werte anpassen, die anderen rauswerfen. Habe die KW1.

Mal unter Windows probiert mit der Testversion vom Hersteller (Viessmann Vitosoft)...? 

Die läuft einige Zeit als Trial, erlaubt den Test des Adapters - und bringt nebenbei interessante XML-Dateien mit.  Jene unbedingt nach der 1. Inbetriebnahme wegsichern!

Im 2. Schritt kann man mit Python die für die eigene Heizung relevanten Daten extrahieren. Infos dazu gibt es im Netz.
